js/pivx_shielding.js

```diff
@@ -82,10 +84,15 @@ export default class PIVXShielding {
     this.unspentNotes = [];
 
     /**
-     * @type {Map<String, String[]>} A map txid->nullifiers, storing pending transaction.
+     * @type {Map<String, {nullifiers: String[], isShield: bool}>} A map txid->nullifiers, storing pending transaction.
      * @private
      */
     this.pendingTransactions = new Map();
+
+    /**
+     * @type {{txid: String, vout: Number, amount: Number, privateKey: Uint8Array, script: Uint8Array}[]}
+     */
+    this.utxos = [];
```

Comment:
this is useelss and we can pass utxos directly inside the createTransaction function

Reply:
After some internal discussion, we've decided to remove this functionality and add it if needed in the future